build.rs

```diff
@@ -16,5 +16,7 @@ fn main() {
         "has_to_int_unchecked",
     );
 
+    ac.emit_path_cfg("u32::reverse_bits()", "has_reverse_bits");
```

Comment:
This test is failing (see `cargo check -vv`) because it will try `pub use PATH;` which doesn't work for inherent methods, nevermind the parentheses. You could try an expression instead, like:
```suggestion
    ac.emit_expression_cfg("1u32.reverse_bits()", "has_reverse_bits");
```

Reply:
Ah, thanks! Good to know that making cargo more noisy outputs some autocfg information.

I don't know how rust libraries work in detail, but the only change I can see in `libnum_traits.rlib` with and without the fallback is in `lib.rmeta`, not sure if that's ok.